environments/extractDiffs.toml

```diff
@@ -52,9 +52,13 @@
         deployed = 11048869
+    [contract.MCD_FLIP_GUSD_A_1_1_5]
```

Comment:
Just remembered that this environment now uses this config to setup the diffs subscription with Geth, so maybe we want to remove this address until the backfill events process has completed (so that we can generate keys for diffs)?